src/javascripts/components/mobile-navigation.js

```diff
@@ -63,6 +63,17 @@ MobileNav.prototype.bindUIEvents = function () {
   })
 }
 
+MobileNav.prototype.addHeadings = function () {
+  var $headings = this.$nav.querySelectorAll('.js-app-mobile-nav-subnav__link-heading')
+
+  nodeListForEach($headings, function ($headingText) {
+    const heading = document.createElement('h3')
```

Comment:
If heading is an element, should it also get the `$` prefix?

```suggestion
    const $heading = document.createElement('h3')
```

Reply:
We also want to use `var` rather than `const` here:

```suggestion
    var heading = document.createElement('h3')
```